.github/workflows/kernel-security-analysis-pr.yml

```diff
@@ -44,9 +44,11 @@ jobs:
            path: kconfig-hardened-check
 
        - name: Check kernel config for security issues
+         # Run kernel-hardening-checker for each kernel config file excluding RISC-V configs, since they are not supported yet.
+         # See https://github.com/a13xp0p0v/kernel-hardening-checker/issues/56
          run: |
            for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
-               if [[  "${file}" = config/kernel/*.config ]]; then
+               if [[ "${file}" = config/kernel/*.config && ! "${file}" =~ riscv ]]; then
```

Comment:
This won't be sufficient. We need to look into the file. This is also riscv:
https://github.com/armbian/build/blob/main/config/kernel/linux-thead-current.config#L5

Reply:
> Oh. That's so obvious.

Obvious is very easy to miss ;) 

OK, lets merge this.